src/base.css

```diff
@@ -29,7 +29,6 @@
 
 
 body {
-	background: linear-gradient(296.57deg, #FF7676 0%, #F54EA2 100%);
```

Comment:
Can we background it to the transction section's dark blue otherwise there's a white bit at the bottom after the content finishes

Reply:
Added